moped-editor/src/views/projects/projectView/NoteInputQuill.js

```diff
@@ -61,11 +83,22 @@ const CommentInputQuill = ({
             />
           </Box>
         </Grid>
+        {!editingNote && (
```

Comment:
oooh it would be nice to be able to edit the note the type. do you think that's reasonable to include in this work? or i'm happy to open a separate issue.

Reply:
i think its reasonable!